plugins/WorldHub/scripts/worldHub.js

```diff
@@ -3,15 +3,22 @@ const { create } = await require('@bridge/sidebar')
 const { WorldHub } = await require('@bridge/ui')
 const { register, addTabActions } = await require('@bridge/tab-actions')
 const { setup } = await require('@bridge/com-mojang')
-const { readdir, getFileHandle, loadFileHandleAsDataUrl, directoryExists } =
-	await require('@bridge/fs')
+const {
+	readdir,
+	getFileHandle,
+	loadFileHandleAsDataUrl,
+	directoryExists,
+	readJSON
+} = await require('@bridge/fs')
 const { getCurrentProject } = await require('@bridge/env')
+const Config = await readJSON(`${getCurrentProject()}/config.json`)
 
 class WorldHubTab extends ContentTab {
 	type = 'WorldHubTab'
 	component = WorldHub
 	availableWorlds = null
 	isReady = false
+	worldsDir = Config?.WorldHub?.dirPath || 'worlds'
```

Comment:
I would prefer the "WorldHub" namespace property to start with a lowercase letter to align it with the other properties within the project config -> Use "worldHub" instead